src/form-sections/cancel.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  PropertyValues,
+  nothing,
+  css,
+} from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import type { MonthlyPlan } from '../models/plan';
+
+import type { IauxButton } from '../presentational/ia-button';
+import '@internetarchive/donation-form/dist/src/form-elements/contact-form/contact-form.js';
+
+import '@internetarchive/donation-form-section';
+
+@customElement('ia-mgc-cancel-plan')
+export class IauxMgcCancelPlan extends LitElement {
+  @property({ type: Object }) plan?: MonthlyPlan;
+
+  @property({ type: Boolean, reflect: true }) patronWantsToKeepPlan = true;
+
+  @property({ type: Boolean, reflect: true }) initialCancelRequest = false;
+
+  @query('form') form!: HTMLFormElement;
+
+  updated(changed: PropertyValues) {
+    if (changed.has('plan')) {
+      console.log('plan updated', this.plan);
+    }
+  }
+
+  async cancelThisPlan(e: Event) {
+    e.preventDefault();
+    this.patronWantsToKeepPlan = false;
+    this.dispatchEvent(new Event('cancelPlan'));
+  }
+
+  get formId(): string {
+    return `cancel-donation-form-${this.plan?.id}`;
+  }
+
+  protected render() {
```

Comment:
Are we not showing the plan info as a form when you click Manage this monthly donation?

Reply:
we are. that is as expected